src/networks.json

```diff
@@ -2118,5 +2118,20 @@
     },
     "start": 38066,
     "logo": "ipfs://QmYGxmEhV6djksUk97pdE9TmL6DXm9KW8BP7pwUv8pqp8r"
+  },
+  "119": {
+    "key": "119",
+    "name": "ENULS Mainnet",
+    "chainId": 119,
+    "network": "mainnet",
+    "multicall": "0xd7d2Aa52f5491cB60ccD85b8c39935BF0baCd142",
+    "rpc": [
+      "https://evmapi.nuls.io/"
```

Comment:
Hi @MIMIEYES this is not a archive node, it return error `missing trie node`

Reply:
> 

Hi @ChaituVR we have changed it to an archive node, pls check.